Description: Use the correct locale.
 The test was expecting a specific locale. Everything seems to indicate
 that any UTF-8 enabled locale would work for this testcases, so use
 C.UTF-8 and avoid to have to depend on all locales.
Author: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Forwarded: no
Last-Update: 2014-01-14

--- a/kioslave/trash/tests/testtrash.cpp
+++ b/kioslave/trash/tests/testtrash.cpp
@@ -58,7 +58,7 @@ int initLocale()
 {
 #ifdef UTF8TEST
     // Assume utf8 system
-    setenv( "LC_ALL", "en_US.utf-8", 1 );
+    setenv( "LC_ALL", "C.UTF-8", 1 );
     setenv( "KDE_UTF8_FILENAMES", "true", 1 );
 #else
     // Ensure a known QFile::encodeName behavior for trashUtf8FileFromHome
